PerfTools/AllocMonitor/scripts/edmModuleAllocJsonToCircles.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+import json
+transitionTypes = ["construction", "event",
+                        "begin job", "begin stream",
+                        "global begin run", "global begin luminosity block",
+                        "stream begin run", "stream begin luminosity block",
+                        ]
+allocTypes = ["added", "nAlloc", "nDealloc", "maxTemp", "max1Alloc"]
+
+def processModuleTransition(moduleLabel, moduleType, moduleInfo, transitionType, moduleTransition):
+        moduleTransition[moduleLabel] = {"cpptype": moduleType, "allocs": []}
+        for entry in moduleInfo:
+            if entry["transition"] == transitionType:
+                moduleTransition[moduleLabel]["allocs"].append(entry.get("alloc",{}))
+        moduleTransition[moduleLabel]["nTransitions"] = len(moduleTransition[moduleLabel]["allocs"])
+
+def formatToCircles(moduleTransitions):
+    modules_dict = {}
+    doc = {
+       "modules": [],
+       "resources": [],
+       "total": {}
+    }
+    for transitionType in transitionTypes:
+        doc["resources"] += [
+            {
+               "name": "added %s" % transitionType,
+               "description": "added memory per %s transition" % transitionType,
+               "title": "Amount of memory added to the process at the end of the %s transition" % transitionType,
+               "unit": "kB"
+            },
+            {
+
+               "name": "nAlloc %s" % transitionType,
+               "description": "num allocs per %s transition" % transitionType,
+               "title": "Number of allocations during the %s transition" % transitionType,
+               "unit": ""
+            },
+            {
+               "name": "max1Alloc %s" % transitionType,
+               "description": "maximum one time allocation per %s transition" % transitionType,
+               "title": "Maximum one time allocation held during the %s transition" % transitionType,
+               "unit": "kB"
+           },
+           {
+               "name": "nDealloc %s" % transitionType,
+               "description": "num deallocs for %s transition" % transitionType,
+               "title": "Number of deallocations during the %s transition" % transitionType,
+               "unit": ""
+           },
+           {
+               "name": "maxTemp %s" % transitionType,
+               "description": "maximum temporary memory per %s transition" % transitionType,
+               "title": "Maximum temporary memory held during the %s transition" % transitionType,
+               "unit": "kB"
+           },
+        ]
+# The circles code uses the "events" field to normalize the values between files with different number of events
+# Here we set it to 1 for the total events because the total is already normalized per transition
```

Comment:
Could you indent the comment?
```suggestion
        # The circles code uses the "events" field to normalize the values between files with different number of events
        # Here we set it to 1 for the total events because the total is already normalized per transition
```

Reply:
Done